Python Examples/TestMonitor/results.py

```diff
@@ -0,0 +1,295 @@
+    """
+    return {
+        "results":results
+    }
+
+
+def test_step_create_or_update_request_object(steps, update_result_total_time=True):
+    """
+    Creates a create/update test result request object 
+    dictionary required for creating the new/update existing test steps.
+    :param results: List of steps that needs to create/update
+    :param update_result_total_time: A boolean to state 
+    whether to update result total time or not
+    :return: A dictionary which is required for creating/updating the steps
+    """
+    return{
+        "steps": steps,
+        "updateResultTotalTime": update_result_total_time
+    }
+
+def update_test_results_request(results, determine_status_from_steps=True):
+    """
+    Creates a update test result request object 
+    dictionary required for updating the existing test results.
+    :param results: List of results that needs to be updated
+    :param determine_status_from_steps: A boolean representing 
+    whether the status of result should be updated based on result or not
+    :return: A dictionary which is required for updating the results
+    """
+    return{
+        "results": results,
+        "determineStatusFromSteps": determine_status_from_steps
+    }
+
+def print_usage_and_exit(error:str):
+    print("This example requires a configuration.")
+    print(error)
+    print()
+    print("Please specify a configuration using the following arguments:")
+    print()
+    print("\t <url> <api_key>")
+    print()
+    print("To run the example against a SystemLink Enterprise, the URL should include the")
+    print("scheme, host, and port if not default. For example:")
+    print("python <example_filename.py> https://myserver:9091 api_keynjnjnjnjnvgcycy")
+    quit()
+
+def main():
+    
+    args = sys.argv
+
+    if len(args) == 3 :
+        host =args[1]
+        api_key = args[2]
+    else:
+        print_usage_and_exit("Please pass all the required arguments")
+
+    create_results_host = f"{host}nitestmonitor/v2/results"
+    create_steps_host = f"{host}nitestmonitor/v2/steps"
+    update_results_host = f"{host}nitestmonitor/v2/update-results"
+    update_steps_host = f"{host}nitestmonitor/v2/update-steps"
+    headers = { 'X-NI-API-KEY': api_key }
+
+    # Set test limits
+    low_limit = 0
+    high_limit = 70
+
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+        "statusType": "RUNNING",
+        "statusName": "Running"
+        },
+        "systemId": None,
+        "hostName": None,
+        "properties":None,
+        "serialNumber": str(uuid.uuid4()),
+        "operator": "John Smith",
+        "partNumber": "NI-ABC-123-PWR1",
+        "fileIds":None,
+        "startedAt": str(datetime.datetime.now()),
+        "totalTimeInSeconds": 0.0
+    }
+
+    create_results_request = create_test_result_request(results=[test_result])
+    request_response = requests.post(create_results_host, json=create_results_request, headers=headers)
```

Comment:
Can we have a separate file and move all the API calls to that?

Reply:
created the post request in the client and calling the APIs from there